src/renderer/components/badge/badge.scss

```diff
@@ -1,14 +1,17 @@
 .Badge {
   display: inline-block;
-  background: $colorVague;
-  color: $textColorSecondary;
-  border-radius: $radius;
-  padding: .2em .4em;
   white-space: nowrap;
   max-width: 100%;
   overflow: hidden;
   text-overflow: ellipsis;
 
+  &:not(.flat) {
```

Comment:
Why `:not`? :) 
Logically we have to define some "view modifier" here with `&.flat { ... }`

Reply:
Will replace these styles in upcoming PR.